docs/contracts/node-operators-registry.md

```diff
@@ -14,126 +39,306 @@ Returns the rewards distribution proportional to the effective stake for each no
-| -------------- | --------- | ---------------- |
-| `_operator_id` | `uint256` | Node Operator id |
+| Name              | Type      | Description      |
+| ----------------- | --------- | ---------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id |
 
 ### getUnusedSigningKeyCount()
 
-Returns number of usable signing keys of the node operator
+Returns the number of usable signing keys of the node operator.
 
 ```sol
-function getUnusedSigningKeyCount(uint256 _operator_id) returns (uint256)
+function getUnusedSigningKeyCount(uint256 _nodeOperatorId) returns (uint256)
 ```
 
-#### Parameters:
+#### Parameters
 
-| Name           | Type      | Description      |
-| -------------- | --------- | ---------------- |
-| `_operator_id` | `uint256` | Node Operator id |
+| Name              | Type      | Description      |
+| ----------------- | --------- | ---------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id |
 
 ### getSigningKey()
 
-Returns n-th signing key of the node operator
+Returns n-th signing key of the node operator.
 
 ```sol
-function getSigningKey(uint256 _operator_id, uint256 _index) returns (
-  bytes key,
-  bytes depositSignature,
-  bool used
+function getSigningKey(uint256 _nodeOperatorId, uint256 _index) returns (
+    bytes key,
+    bytes depositSignature,
+    bool used
 )
 ```
 
-#### Parameters:
+#### Parameters
 
-| Name           | Type      | Description                        |
-| -------------- | --------- | ---------------------------------- |
-| `_operator_id` | `uint256` | Node Operator id                   |
-| `_index`       | `uint256` | Index of the key, starting with 0d |
+| Name              | Type      | Description                       |
+| ----------------- | --------- | --------------------------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id                  |
+| `_index`          | `uint256` | Index of the key, starting with 0 |
 
-#### Returns:
+#### Returns
 
 | Name               | Type    | Description                                           |
 | ------------------ | ------- | ----------------------------------------------------- |
 | `key`              | `bytes` | Key                                                   |
 | `depositSignature` | `bytes` | Signature needed for a `depositContract.deposit` call |
 | `used`             | `bool`  | Flag indication if the key was used in the staking    |
 
+### getSigningKeys()
+
+Returns `_limit` signing keys of the node operator.
+
+```sol
+function getSigningKey(uint256 _nodeOperatorId, uint256 _offset, uint256 _limit) returns (
+    bytes pubkeys,
+    bytes signatures,
+    bool[] used
+)
+```
+
+#### Parameters
+
+| Name              | Type      | Description                        |
+| ----------------- | --------- | ---------------------------------- |
+| `_nodeOperatorId` | `uint256` | Node Operator id                   |
+| `_offset`         | `uint256` | Offset of the key, starting with 0 |
```

Comment:
Sorry, but what is offset? What unit it measured in? Offset in where? Is it an index in array or something? :shrug: